wave_vision/include/wave/vision/matcher/flann_matcher.hpp

```diff
@@ -231,7 +245,7 @@ class FLANNMatcher : public DescriptorMatcher {
       cv::Mat &descriptors_2,
       const std::vector<cv::KeyPoint> &keypoints_1,
       const std::vector<cv::KeyPoint> &keypoints_2,
-      cv::InputArray mask = cv::noArray()) const override;
+      cv::InputArray mask = cv::noArray()) override;
```

Comment:
Maybe add virtual at the beginning of the function declartion?

Reply:
okay I did some searching. I don't think this is necessary. The virtual keyword I've used in the base class, and now I'm overriding it here.

It looks like you can write virtual and there is no change in behaviour but I feel like that just adds confusion?

Reply:
virtual in the derived class is purely stylistic.just more explicit.